base/applications/mmc/console.c

```diff
@@ -334,6 +345,12 @@ FrameOnCommand(HWND hwnd,
             PostMessage(hwnd, WM_CLOSE, 0, 0);
             break;
 
+        case IDM_HELP_ABOUT:
+            AllocAndLoadString(&lpTitle, hAppInstance, IDS_APPTITLE);
```

Comment:
Why no failure handling here, compared to the other change?

Reply:
Added